app/mailers/user_session_mailer.rb

```diff
@@ -1,6 +1,7 @@
 class UserSessionMailer < SchoolMailer
   def send_login_token(user, url_options)
     @user = user
+    user.update!(login_mail_sent_at: Time.zone.now)
```

Comment:
@prashanth726 Any particular reason why the time is updated here as well? The `MailLoginTokenService` which consumes this mailer already handles this. I am guessing it is for the mailer preview to work?